tests/installation/releasenotes_online.pm

```diff
@@ -0,0 +1,37 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017-2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Check origin of displayed releasenotes during installation
+# Maintainer: mgriessmeier <mgriessmeier@suse.com>, Nick Singer <nsinger@suse.de>
```

Comment:
For new tests, please borrow a 'header' that only contains copyright 2021 and Mantainer, our team, Tags is not needed, just copy/paste from another example.